### DISCUSSION TITLE
"Because you follow #tag" on dashboard

### DISCUSSION THREAD

**Discussion by evuie** [AprilSylph/XKit-Rewritten#1645] [Suggestion Box]
Would it be possible to tweak this function in any way? I'd love to use it to discover new art in the fandoms I follow but the core tumblr feature will just show me the same post over and over until I hit "not interested" 
If there's any way to prevent the constant repeat loop, maybe options to not suggest posts you've already reblogged that would be amazing ^__^

**Comment by AprilSylph** [comment]
XKit can't control Tumblr's recommendation engine, nor is it remotely feasible to create a replacement recommendation service.
